app/models/concerns/cas_client_data.rb

```diff
@@ -92,7 +96,7 @@ def self.ignored_for_batch_maintenance
     end
 
     def self.manual_cas_columns
-      cas_columns.except(:hiv_positive, :dmh_eligible, :chronically_homeless_for_cas, :full_housing_release, :limited_cas_release, :housing_release_status, :sync_with_cas, :hues_eligible, :disability_verified_on, :required_number_of_bedrooms, :required_minimum_occupancy, :cas_match_override, :health_prioritized, :tie_breaker_date).
+      cas_columns.except(:hiv_positive, :dmh_eligible, :chronically_homeless_for_cas, :full_housing_release, :limited_cas_release, :housing_release_status, :sync_with_cas, :hues_eligible, :disability_verified_on, :required_number_of_bedrooms, :required_minimum_occupancy, :cas_match_override, :health_prioritized, :tie_breaker_date, :vispdat_length_homeless_in_days).
```

Comment:
I needed this change to get the default calculator working . it was throwing an error on the cas readiness page:
```
undefined method `vispdat_length_homeless_in_days' for #<GrdaWarehouse::Hud::Client:0x0000ffff95ecf258>
Did you mean?  vispdat_prioritization_days_homeless
```
what's the correct fix for that?

Reply:
I believe that was correct.  Thanks!